Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  "pkgconfig(gio-unix-2.0)" \
+  "pkgconfig(glib-2.0) >= 2.31.0" \
+  "pkgconfig(gtk+-3.0)" \
+  "pkgconfig(libgnome-menu-3.0)"
```

Comment:
again adding some more packages into built image like in other PR. Dependency issue? Pls add a comment explaining this.